test/shared/forkHelpers.ts

```diff
@@ -33,6 +33,13 @@ export function impersonate(accounts: string[]) {
   );
 }
 
+export async function setBalance(address: string, balance: BigNumber) {
+  await network.provider.send('hardhat_setBalance', [
```

Comment:
you don't need `async` nor `await`, you can just put a return in here, should accomplish the same

Reply:
[fixed](https://github.com/lindy-labs/sc_solidity-contracts/pull/171/commits/399ba46c154f211f1659af975896488d653f1e29)